src/items.rs

```diff
@@ -1813,11 +1813,21 @@ fn rewrite_static(
         let remaining_width = context.budget(offset.block_indent + 1);
         rewrite_assign_rhs(
             context,
-            lhs,
+            &lhs,
             &**expr,
             Shape::legacy(remaining_width, offset.block_only()),
         )
         .and_then(|res| recover_comment_removed(res, static_parts.span, context))
+        .or_else(|| {
+            let nested_indent = offset.block_indent(context.config);
+            let rhs = context.snippet(expr.span).trim();
+            Some(format!(
+                "{}{}{}",
+                lhs,
+                nested_indent.to_string_with_newline(context.config),
```

Comment:
We should add a newline only if the original code has a newline between `=` and the rhs so that we don't get a weird indentation on the rhs.

Reply:
Thank you for the review. I fixed it.